pkgs/by-name/wo/wordlists/package.nix

```diff
@@ -1,45 +1,63 @@
 { lib
-, callPackage
+, symlinkJoin
 , nmap
 , rockyou
-, runtimeShell
 , seclists
-, symlinkJoin
-, tree
 , wfuzz
 , lists ? [
     nmap
     rockyou
     seclists
     wfuzz
   ]
+, writeShellScriptBin
+, tree
 }:
+let
+  version = "unstable-2023-11-14";
 
-symlinkJoin rec {
-  pname = "wordlists";
-  version = "unstable-2023-10-10";
-
-  name = "${pname}-${version}";
-  paths = lists;
+  wordlistsCollection = symlinkJoin {
+    name = "wordlists-collection-${version}";
+    paths = lists;
 
-  postBuild = ''
-    mkdir -p $out/bin
+    postBuild = let
+      pathsToKeep = lib.pipe [
+        "$out/share/wordlists/*"
+        "$out/share/wordlists"
+        "$out/share"
+        "$out"
+      ] [
+        (map (p: ''-path "${p}"''))
+        (builtins.concatStringsSep " -o ")
+      ];
+    in ''
+      find $out -type d ! \( ${pathsToKeep} \) -printf "rm -r \"%p\" 2>/dev/null || true\n" | sh
+    '';
```

Comment:
```suggestion
    postBuild = ''
      shopt -s extglob
      rm -rf $out/!(share)
      rm -rf $out/share/!(wordlist)
      shopt -u extglob
    '';
```

Reply:
https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Pattern-Matching

Reply:
Thank you!

Reply:
That looks a lot better thank you.